src/main/java/org/carlmontrobotics/subsystems/Drivetrain.java

```diff
@@ -1008,6 +1031,41 @@ public void keepRotateMotorsAtDegrees(int angle) {
         }
     }
 
+    // pose estimator stuff
+
+    public void updateMT2PoseEstimator() {
+        boolean rejectVisionUpdate = false;
+
+        LimelightHelpers.SetRobotOrientation(SHOOTER_LL_NAME,
+                poseEstimator.getEstimatedPosition().getRotation().getDegrees(), 0, 0, 0, 0, 0);
+        LimelightHelpers.PoseEstimate visionPoseEstimate = LimelightHelpers
+                .getBotPoseEstimate_wpiBlue_MegaTag2(SHOOTER_LL_NAME);
+
+        if (Math.abs(getGyroRate()) > MAX_TRUSTED_ANG_VEL_DEG_PER_SEC) { // degrees per second
+            rejectVisionUpdate = true;
+        }
+
+        if (visionPoseEstimate.tagCount == 0) {
+            rejectVisionUpdate = true;
+        }
+
+        if (!rejectVisionUpdate) {
+            poseEstimator
+                    .setVisionMeasurementStdDevs(
+                            VecBuilder.fill(STD_DEV_X_METERS, STD_DEV_Y_METERS, STD_DEV_HEADING_RADS));
```

Comment:
Are we tuning these values during testing? If so, you probably want to make them easily editable.